BUILD.md

```diff
@@ -299,6 +299,13 @@ sudo apt-get install ninja-build gettext cmake unzip curl build-essential
 sudo dnf -y install ninja-build cmake gcc make unzip gettext curl
 ```
 
+### Fedora 39
+
+```
+sudo dnf -y install ninja-build cmake gcc make unzip gettext curl && \
+sudo dnf groupinstall -y "Development Libraries"
+```
+
```

Comment:
Right above this are Fedora requirements why make them seperate instead of updating existing ones?